Makefile

```diff
@@ -209,7 +209,7 @@ xgo:
 install-os-dependencies:
 	_assets/scripts/install_deps.sh
 
-setup: install-os-dependencies dep-install lint-install mock-install deploy-install gen-install update-fleet-config ##@other Prepare project for first build
+setup: install-os-dependencies dep-install lint-install mock-install deploy-install gen-install update-fleet-config xgo ##@other Prepare project for first build
```

Comment:
The problem with this is that `setup` is designed for development. `xgo` is pretty heavy (downloading a docker image which weights over 4GB). 

I would just add it in `Makefile`:
```
    stage('Setup') { steps { dir(env.STATUS_PATH) {
      sh 'make setup'
      sh 'make xgo'
    } } }
```

Or introduce a new target in Makefile `setup-build-tools: xgo`.

Reply:
Yeah, I was thinking about doing this at first. Good suggestion.